recipes/pytest-datadir/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "pytest-datadir" %}
+{% set version = "0.2.0" %}
+{% set hash_type = "sha256" %}
+{% set hash = "609d9da4f330e615e97918a03909b2905f92e01f5294708d8fd7ff2b47267e9e" %}
+{% set ext = "zip" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.{{ ext }}
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ ext }}
+  {{ hash_type }}: {{ hash }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - pytest
+
+test:
+  requires:
+    - pytest
+  imports:
+    - pytest_datadir
+  commands:
+    - py.test --help
+
+about:
+  home: https://github.com/gabrielcnr/pytest-datadir
+  license: MIT
+  summary: pytest plugin for manipulating test data directories and files.
+
+extra:
+  recipe-maintainers:
+    - nicoddemus
+    - gabrielcnr
+    - edisongustavo
+    - gqmelo
```

Comment:
Just to verify everyone is ok being a maintainer, quick poll of interest @gabrielcnr @edisongustavo @gqmelo ?